roles/reproducer/tasks/configure_controller.yml

```diff
@@ -21,7 +21,12 @@
   delegate_to: controller-0
   delegate_facts: false
   vars:
+    cifmw_sushy_emulator_hypervisor_target: "{{ inventory_hostname }}"
+    cifmw_sushy_emulator_install_type: podman
+    cifmw_sushy_emulator_hypervisor_target_connection_ip: "{{ inventory_hostname }}.utility"
+    # cifmw_sushy_emulator_hypervisor_target_connection_ip: "{{ network_bridge_info['cifmw-public'] }}"
     cifmw_sushy_emulator_basedir: "{{ _ctl_reproducer_basedir }}"
+    cifmw_sushy_emulator_listen_ip: "controller-0.utility"
```

Comment:
By default it listens on all IPs, I don't think we need to set the listen_ip here.